src/multi_select.coffee

```diff
@@ -47,10 +47,27 @@ Ember.Widgets.MultiSelectComponent = Ember.Widgets.SelectComponent.extend
     @get('placeholder') or @get('persistentPlaceholder')
   .property 'query', 'placeholder', 'persistentPlaceholder', 'selections.length'
 
+  # add css class to multiselect to highlight border
```

Comment:
that said, `addBorderToMultiSelectContainer` and `removeBorderFromMultiSelectContainer` would be better